packages/components/src/utils/dom/getShadowRootHTMLElements.ts

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable prefer-arrow/prefer-arrow-functions */
+import {getHTMLElements} from './getHTMLElements';
```

Comment:
Looks like this isn't formatted with prettier.
Also what about unit tests?

Reply:
I formatted it and added a test for getHTMLElements